packages/components/src/components/data-table/table-row-info.tsx

```diff
@@ -15,20 +17,41 @@ type TTableRowIndex = {
 const TableRowInfo = ({ replace, is_footer, cells, className, is_dynamic_height, measure }: TTableRowIndex) => {
     const [show_details, setShowDetails] = React.useState(false);
 
+    const debouncedHideDetails = React.useMemo(
```

Comment:
The code is repeated here, can we create a hook maybe? we can return `deboucehidedetails` and `toggledetailsdecorator` from the hook

Reply:
Good catch, let me try and think)

Reply:
Applied, thank you)
(except toggleDetailsDecorator)